examples/js/archive.js

```diff
@@ -213,29 +216,28 @@ function isJsonDetected(url) {
 function placeImage (imageURL, options, newImage = false) {
   let image;
   
-  if (newImage) { 
+  if (newImage) { // Construct new map
     image = L.distortableImageOverlay(
       imageURL,
       {tooltipText: options.tooltipText}
     );
-  } else {
+  } else { // Reconstruct map to previous saved state
     image = L.distortableImageOverlay(
       imageURL,
       {
-        height: options.height,
```

Comment:
I removed height here because it'd be auto-detected if we do the fitBounds thing.

Reply:
Interesting... okay.